clustering/kubernetes.py

```diff
@@ -330,7 +330,10 @@ def main():
     file_reference = module.params.get('file_reference')
 
     if inline_data:
-        data = inline_data
+        if not isinstance(inline_data, dict) and not isinstance(inline_data, list):
+            data = yaml.load(inline_data)
+        else:
+            data = inline_data
```

Comment:
@bcoca I see a jsonarg type, but that appears to expect to keep something as a json'ified string rather than convert from json to a python data structure, and it also doesn't appear to handle yaml. Am I missing something?